scripts/src/javatests/org/oppia/android/scripts/kdoc/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+"""
+Tests corresponding to kdoc presence check, that ensures all public members (except overrides) must
+have a corresponding KDoc.
```

Comment:
As mentioned earlier, this docstring is a bit too specific to the exact check being implemented. Please generalize it.

Reply:
Done.